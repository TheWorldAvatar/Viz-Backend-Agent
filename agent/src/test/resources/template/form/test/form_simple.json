[
  {
    "@id": "string_id",
    "@type": [
      "http://www.w3.org/ns/shacl#PropertyShape"
    ],
    "http://www.w3.org/ns/shacl#name": [
      {
        "@value": "name"
      }
    ],
    "http://www.w3.org/ns/shacl#description": [
      {
        "@value": "The first name of a person"
      }
    ],
    "http://www.w3.org/ns/shacl#datatype": [
      {
        "@id": "http://www.w3.org/2001/XMLSchema#string"
      }
    ],
    "http://www.w3.org/ns/shacl#order": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": "1"
      }
    ]
  },
  {
    "@id": "dec_id",
    "@type": [
      "http://www.w3.org/ns/shacl#PropertyShape"
    ],
    "http://www.w3.org/ns/shacl#name": [
      {
        "@value": "age"
      }
    ],
    "http://www.w3.org/ns/shacl#description": [
      {
        "@value": "The age of a person"
      }
    ],
    "http://www.w3.org/ns/shacl#datatype": [
      {
        "@id": "http://www.w3.org/2001/XMLSchema#integer"
      }
    ],
    "http://www.w3.org/ns/shacl#order": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": "2"
      }
    ]
  }
]