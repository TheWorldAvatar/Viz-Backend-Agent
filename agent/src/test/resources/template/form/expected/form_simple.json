{
  "node": [],
  "property": [
    {
      "datatype": "string",
      "@type": "http://www.w3.org/ns/shacl#PropertyShape",
      "name": {
        "@value": "id"
      },
      "description": {
        "@value": "ID"
      },
      "minCount": {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": "1"
      },
      "@id": "string_id",
      "maxCount": {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": "1"
      },
      "order": -1
    },
    {
      "datatype": "string",
      "@type": "http://www.w3.org/ns/shacl#PropertyShape",
      "name": {
        "@value": "name"
      },
      "description": {
        "@value": "The first name of a person"
      },
      "@id": "string_id",
      "order": 1
    },
    {
      "datatype": "integer",
      "@type": "http://www.w3.org/ns/shacl#PropertyShape",
      "name": {
        "@value": "age"
      },
      "description": {
        "@value": "The age of a person"
      },
      "@id": "dec_id",
      "order": 2
    }
  ],
  "@context": {
    "PropertyShape": "http://www.w3.org/ns/shacl#PropertyShape",
    "PropertyGroup": "http://www.w3.org/ns/shacl#PropertyGroup",
    "in": "http://www.w3.org/ns/shacl#in",
    "defaultValue": "http://www.w3.org/ns/shacl#defaultValue",
    "description": "http://www.w3.org/ns/shacl#description",
    "label": "http://www.w3.org/2000/01/rdf-schema#label",
    "node": "http://www.w3.org/ns/shacl#node",
    "datatype": "http://www.w3.org/ns/shacl#datatype",
    "name": "http://www.w3.org/ns/shacl#name",
    "property": "http://www.w3.org/ns/shacl#property",
    "comment": "http://www.w3.org/2000/01/rdf-schema#comment",
    "class": "http://www.w3.org/ns/shacl#class",
    "order": "http://www.w3.org/ns/shacl#order",
    "group": "http://www.w3.org/ns/shacl#group"
  }
}